docs/GettingStartedDocs/Contributors/InstallInfo.md

```diff
@@ -64,3 +64,14 @@ as above. For example, to create a Debian package that will install the SDK to
 cmake -DCMAKE_INSTALL_PREFIX=/opt/openenclave ..
 cpack -G DEB
 ```
+
+## Create the Host-only Report Verification package
```

Comment:
Copyedit: Is "Host-only Report Verification" package a proper noun? I would follow the existing casing convention and go with:
> ## Create the host-only report verification package

Reply:
Fixed